locales/en/messages.json

```diff
@@ -3741,6 +3747,12 @@
     "pidTuningDynamicNotchMaxHzHelp": {
         "message": "Set this to the highest incoming noise frequency that is needed to be controlled by the dynamic notch."
     },
+    "pidTuningDynNotchCountHelp": {
```

Comment:
Is strange to use `pidTuningDynamicNotch...` across all the code but for this new values we have only `pidTuningDynNotch...` I think is better to use unified names, here and in the javascript/html code.

Reply:
Yeah i have based on Firmware names this time, fixed.